libraries/botbuilder-dialogs-adaptive/src/input/textInput.ts

```diff
@@ -37,7 +37,9 @@ export class TextInput extends InputDialog implements TextInputConfiguration {
 
         if (this.outputFormat) {
             const value = this.outputFormat.getValue(dc.state);
-            input = value.toString();
+            if (value && value.trim()) {
+                input = value.toString();
```

Comment:
It seems strange to call `toString()` after calling `trim()` which implies `value` is a string. Also, do we want to assign the trimmed string to `input`? Maybe this should be:
```typescript
const maybeValue = this.outputFormat.getValue(dc.state);
if (maybeValue && typeof maybeValue === 'string') {
  const value = maybeValue.trim();
  if (value) {
    input = value;
  }
}
```

Reply:
Yes. The original `toString()` is redundant because the return type of `this.outputFormat.getValue(dc.state);` is string | undefined. So, do you think just remove `toString()` is ok?

Reply:
Yes. The original `toString()` seems redundant because the return type of `this.outputFormat.getValue(dc.state)` is string | undefined, so, just remove the `toString()` is ok?